.evergreen/config.in.yml

```diff
@@ -882,6 +882,23 @@ functions:
         binary: bash
         args:
           - ${PROJECT_DIRECTORY}/.evergreen/run-benchmarks.sh
+
+  # TEMP!
```

Comment:
TODO with linked ticket to remove this task / consolidate with the other tasks?

Reply:
done